src/main/java/edu/wpi/first/gradlerio/GradleRIOPlugin.java

```diff
@@ -56,9 +57,14 @@ interface Parameters extends FlowParameters {
         @Override
         public void execute(Parameters parameters) {
             Optional<Throwable> failure = parameters.getBuildResult().get().getFailure();
-            if (System.getProperty("user.dir").contains("OneDrive")) {
-                throw new Error("Cannot use project inside OneDrive");
+            try {
+                if (System.getProperty("user.dir").contains("OneDrive")) {
+                    throw new OneDriveException();
+                }
+            } catch(OneDriveException e) {
```

Comment:
This is just going to completely suppress the warning. Extend the exception from RuntimeException and just let it get thrown all the way up.

Reply:
I wasnt aware I had to use throws to throw an error has been fixed now